src/concrete/ml/quantization/quantized_module.py

```diff
@@ -483,12 +487,22 @@ def _clear_forward(
         # The output of a graph must be a QuantizedArray
         assert all(isinstance(elt, QuantizedArray) for elt in output_quantized_arrays)
 
-        results = tuple(
+        q_results = tuple(
             elt.qvalues for elt in output_quantized_arrays if isinstance(elt, QuantizedArray)
         )
-        if len(results) == 1:
-            return results[0]
-        return results
+
+        if self._composition_mapping is not None:
```

Comment:
this part adds the requant in FHE

Reply:
Unless I miss something important on how the composition works, I don't think we should touch the quantized module.

We could probably add a quantized op in the graph that is created in the post_training.py at calibration. Maybe a QuantizedRequant that uses the model input quantizer as its output quantizer.

Reply:
see my answer here : https://github.com/zama-ai/concrete-ml/pull/660#discussion_r1624431458